src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -646,8 +647,10 @@ namespace Akka.Actor
     }
     public struct Envelope
     {
-        public object Message { get; set; }
-        public Akka.Actor.IActorRef Sender { get; set; }
+        public Envelope(object message, Akka.Actor.IActorRef sender, Akka.Actor.ActorSystem system) { }
```

Comment:
I like this change a lot. Making the setters private and using a constructor instead.